src/logging/logging.ts

```diff
@@ -43,20 +35,45 @@ export class Log {
   constructor(private tag_:string) {
     this.logger = freedom.core().getLogger(this.tag_);
   }
+
+  private log_ = (level :string, args :any[]) :void => {
+    var message :string;
+
+    if (2 === args.length && 'string' === typeof(args[0]) && Array.isArray(args[1])) {
```

Comment:
Huh. I wasn't aware of that.

I guess this is fine for now but I just started a thread on `uproxy-eng`.